backend/src/test/java/de/neuefische/backend/BackendApplicationTests.java

```diff
@@ -6,8 +6,5 @@
 @SpringBootTest
 class BackendApplicationTests {
 
-    @Test
-    void contextLoads() {
-    }
```

Comment:
why deleteing default test?

Reply:
otherwise the tests dont pass. if there is a better solution we gonna fix it

Reply:
no no no. no deleting tests :D u have to put a placeholder env variable into your application.properties within your test directory